system/templates/rolls/preroll-dialog.hbs

```diff
@@ -1,84 +1,117 @@
-{{#*inline "statDescription"}}
-{{#if (eq Using.length 1)}}
-roll +{{lookup Using 0}}
-{{else}}
-roll {{Method}} of {{join Using delimiter=', '}} {{!-- TODO: i18n --}}
-{{/if}}
-{{/inline}}
-
-<form class="preroll-dialog">
+<style>
+  .preroll-dialog { display: flex; flex-direction: column; gap: 5px; }
+</style>
+<form class='preroll-dialog'>
   {{#if move}}
-  <div>{{{enrichMarkdown move.data.data.Trigger.Text}}}</div>
+    <style>
+      .roll-trigger-text { }
+    </style>
```

Comment:
Is it your intent to include global styles in every dialog that's rendered?

Reply:
more just a convenience to keep them together - same principle as SFCs. unfortunately, i discovered that doesnt actually work except for handlebars static sites (obvious in hindsight :sweat_smile:).

i think the ideal solution would be some vite processing to extract the styles themselves - so, source handlebars files live in `src`, then get processed and dumped in system, while the contents of their style tags is included in the standard css file.

a less labor-intensive method of component-izing their specific styling would be something like: there's a preroll-dialog.less file that lives next to this one, and it's imported when the style sheet is build.

short term, i just can move them to the main styles.less file, tho. i'll include this template's name in a comment.